src/jade/config/run_config.py

```
@@ -157,39 +157,43 @@ class EnvironmentVariables:
-                )
-            elif not os.path.exists(self.batch_template):
-                raise ConfigError(f"Batch template {self.batch_template} not found")
-
-            if self.batch_system is None:
-                raise ConfigError(
-                    "Batch system is needed for job submission, please provide one"
-                )
+        if self.run_mode == RunMode.JOB_SUBMISSION and self.scheduler_command is None:
+            raise ConfigError(
+                "Scheduler command is needed if run_mode is 'job', please provide one"
+            )
```

Ruff (0.13.2)

Avoid specifying long messages outside the exception class

(TRY003)